src/lang/po/Prusa-Firmware-Buddy.pot

```diff
@@ -305,264 +267,255 @@ msgstr ""
 msgid "Fan1 RPM"
 msgstr ""
 
-#: src/gui/screen_menu_settings.cpp:24
-msgid "Fil. sens."
+#: src/gui/screen_home.cpp:34 src/guiapi/include/MItem_menus.hpp:29
+msgid "Filament"
 msgstr ""
 
-#: src/gui/screen_home.cpp:44 src/guiapi/include/MItem_menus.hpp:18
-msgid "Filament"
+#: src/gui/screen_menu_settings.cpp:25 src/guiapi/include/MItem_tools.hpp:359
+msgid "Filament Sensor"
 msgstr ""
 
-#: src/gui/screen_print_preview.cpp:320
+#: src/gui/screen_print_preview.cpp:223
 msgid ""
 "Filament not detected. Load filament now? Select NO to cancel, or IGNORE to "
 "disable the filament sensor and continue."
 msgstr ""
 
-#: src/gui/dialogs/DialogG162.cpp:26 src/gui/dialogs/DialogLoadUnload.cpp:39
-msgid ""
-"Finishing         \n"
-"buffered gcodes.  \n"
+#. ***************************************************************************
+#. clang-format off
+#: src/gui/dialogs/DialogG162.cpp:8 src/gui/dialogs/DialogLoadUnload.cpp:14
+msgid "Finishing buffered gcodes."
 msgstr ""
 
 #. r=1 c=20
-#: src/gui/screen_menu_version_info.cpp:64
+#: src/gui/screen_menu_version_info.cpp:60
 msgid "Firmware Version\n"
```

Comment:
```suggestion
msgid "Firmware Version"
```

Reply:
No way, unless you fix the version info screen